app/src/main/java/com/techcafe/todone/App.kt

```diff
@@ -9,9 +11,19 @@ class App : Application() {
 
     override fun onCreate() {
         super.onCreate()
+        setupNightMode()
         startKoin {
             androidContext(this@App)
             modules(modules = moduleList)
         }
     }
+    // 別にこれ実行しなくても端末の設定に応じてダークモードに切り替わる
+    private fun setupNightMode() {
+        val nightMode = if (BuildCompat.isAtLeastQ()) {
```

Comment:
isAtLeastQはDeplicatedで、こっち使うらしいぞ！
```suggestion
        val nightMode = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Reply:
https://github.com/TechCafeFreedom/TODONE/pull/68/commits/26d213f1bb660fc3f478b780e34afc66185ed289 でなおしたんごお